### ISSUE TITLE
[bug] path not implemented

### ISSUE THREAD

**Issue by curtbushko** [eikster-dk/gh-worktree#9]
Awesome extension! Thanks for putting this together!

[path] is listed in the pr command but it isn't implemented. 

```
$> gh worktree pr 3284 /reviews/3284
Usage:
  worktree pr [number] [path] [flags]

Examples:
gh worktree pr 41

Flags:
  -h, --help   help for pr

exit status 128
```

**Comment by eikster-dk** [comment]
Thank you for using it. I'm currently one a week vacation so I won't be able to have a look at it before I'm back.

But I have put it on my to-do so I can look at it when I'm back

**Comment by eikster-dk** [comment]
Fixed in version 0.4.0 :) 

You can upgrade with `gh extension upgrade eikster-dk/gh-worktree`

**Comment by curtbushko** [comment]
Thank you!
